### ISSUE TITLE
[Feature] Add a quick css/overide theme for user changes

### ISSUE THREAD

**Issue by Josh65-2201** [SteamClientHomebrew/Millennium#114]
### Before Requesting

- [X] I found no existing issue matching my feature request
- [X] This request is related to Millennium, and not a request for a specific theme/addon

### Describe the feature you'd like!

There are thing I want to change in a theme but don't always make senses or gets accepted to add to the theme. Using an interface like Vencord/Better discord so we can make the changes that take priority without needing to copy them on every theme update.

**Comment by shdwmtr** [comment]
I agree, and this has been planned for a long time. It might be implemented in the coming updates.

**Comment by ejalxndr** [comment]
Added in new alpha releases
